action_server/src/robocorp/action_server/_server_expose.py

```diff
@@ -26,6 +27,19 @@ class BodyPayload(BaseModel):
     headers: dict
 
 
+def get_expose_session(payload: SessionPayload) -> str:
+    return f"{payload.sessionId}:{payload.sessionSecret}".encode("ascii").hex()
```

Comment:
I started with passing both session secret and id as separate args, but IMHO encoding them in a single token makes for better ergonomics.